packages/google_sign_in/README.md

```diff
@@ -18,41 +18,39 @@ manager](https://console.developers.google.com/). For example, if you
 want to mimic the behavior of the Google Sign-In sample app, you'll need to
 enable the [Google People API](https://developers.google.com/people/).
 
-# iOS integration
+## iOS integration
 
-To access Google Sign-In, you'll need to make sure to [register your
-application](https://developers.google.com/mobile/add?platform=ios). Add
-the generated GoogleService-Info.plist to root of your Runner project in Xcode,
-so that the Google Sign-In framework can determine your client id.
-
-You'll need to add this to the main dictionary of your application's Info.plist:
+1. [First register your application](https://developers.google.com/mobile/add?platform=ios).
+2. Open Xcode. You'll have to paste this into Xcode to properly register `GoogleServices-Info.plist`.
```

Comment:
Good question. That should probably read as paste this code snippet below or something like that.